assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -549,13 +551,20 @@ const MapDisplay = ({
       shapes={[]}
       stateClasses={stateClasses}
     >
-      <SelectionDataLayers
-        selectedEntity={selectedEntity}
-        showSelectionCard={showSelectionCard}
-        deleteSelection={deleteSelection}
-        setSelection={setSelection}
-        setStateClasses={setStateClasses}
-      />
+      <AutoPanPaddingConfig.Provider
+        value={{
+          autoPanPaddingTopLeft: [445, 50],
+          autoPanPaddingBottomRight: [50, 20],
```

Comment:
question: Is `autoPanPaddingBottomRight` meant to match the `paddingBottomRight` on 118?